src/executor/query.rs

```diff
@@ -276,23 +282,21 @@ macro_rules! try_getable_unsigned {
             fn try_get_by<I: ColIdx>(res: &QueryResult, idx: I) -> Result<Self, TryGetError> {
                 match &res.row {
                     #[cfg(feature = "sqlx-mysql")]
-                    QueryResultRow::SqlxMySql(row) => {
-                        use sqlx::Row;
-                        row.try_get::<Option<$type>, _>(idx.as_sqlx_mysql_index())
-                            .map_err(|e| TryGetError::DbErr(crate::sqlx_error_to_query_err(e)))
-                            .and_then(|opt| opt.ok_or_else(|| err_null_idx_col(idx)))
-                    }
+                    QueryResultRow::SqlxMySql(row) => row
+                        .try_get::<Option<$type>, _>(idx.as_sqlx_mysql_index())
+                        .map_err(|e| sqlx_error_to_query_err(e).into())
+                        .and_then(|opt| opt.ok_or_else(|| err_null_idx_col(idx))),
                     #[cfg(feature = "sqlx-postgres")]
-                    QueryResultRow::SqlxPostgres(_) => {
-                        panic!("{} unsupported by sqlx-postgres", stringify!($type))
-                    }
+                    QueryResultRow::SqlxPostgres(_) => Err(query_err(format!(
+                        "{} unsupported by sqlx-postgres",
```

Comment:
This is not a query error, better to be a Type Error

Reply:
https://github.com/SeaQL/sea-orm/pull/1231/commits/b665c429356a2b70bbb727ff267552c2ec926cb4